.github/workflows/tests.yml

```diff
@@ -101,6 +101,7 @@ jobs:
             priv
             react_renderer/dist/app.js
           key: ci-application-cache-${{ github.sha }}
+      - run: npm install --prefix assets -S -install-links deps/mbta_metro/priv/
```

Comment:
📓 This is necessary to install mbta_metro from a path, but getting its deps and putting them in `node_modules`.